src/Resources/config/services.yml

```diff
@@ -18,3 +18,9 @@ services:
             - "@?logger"
         tags:
             - { name: doctrine.event_listener, event: preFlush, priority: 15 }
+
+    entity_tracker.provider.blamable:
+        class: Hostnet\Bundle\EntityTrackerBundle\Services\Blamable
+        arguments:
+            - "@security.token_storage"
+            - "" # It is being injected by the Extension
```

Comment:
Can you move this service definition to `blamable.yml`? `services.yml` is always loaded so it will add this service while it cannot be used without the blamable component.`blamable.yml` is only loaded if the component is installed.